fog/ledger/server/src/key_image_service.rs

```diff
@@ -227,8 +229,23 @@ impl<L: Ledger + Clone, E: LedgerEnclaveProxy> KeyImageStoreApi for KeyImageServ
             if let Err(err) = self.authenticator.authenticate_rpc(&ctx) {
                 return send_result(ctx, sink, err.into(), logger);
             }
+            let start_time = Instant::now();
+
             let response =
                 self.process_queries(self.client_listen_uri.clone(), req.queries.into_vec());
+
+            let status_str = match response.status {
+                MultiKeyImageStoreResponseStatus::UNKNOWN => "UNKNOWN",
+                MultiKeyImageStoreResponseStatus::SUCCESS => "SUCCESS",
+                MultiKeyImageStoreResponseStatus::INVALID_ARGUMENT => "INVALID_ARGUMENT",
+                MultiKeyImageStoreResponseStatus::NOT_READY => "NOT_READY",
+                MultiKeyImageStoreResponseStatus::AUTHENTICATION_ERROR => "AUTHENTICATION_ERROR",
+            };
```

Comment:
I believe this can be replaced with `let status_str = format!("{:?}", response.status)`